docs/WebSocket.md

```diff
@@ -59,9 +83,9 @@ The number of bytes of data that have been buffered (queued but not yet transmit
 
 ##### Instances
 ``` purescript
-Generic BufferedAmount
-Eq BufferedAmount
-Ord BufferedAmount
+instance genericBufferedAmount :: Generic BufferedAmount
+instance eqBufferedAmount :: Eq BufferedAmount
+instance ordBufferedAmount :: Ord BufferedAmount
```

Comment:
Actually, don't worry.

If you build it with latest compiler it would fail because of changes in `generics` (that's why travis build has failed). I'll merge it now, and then would make updates so that it works with newest compiler release.